src/parser/jobs/nin/modules/Ninki.js

```diff
@@ -14,36 +14,51 @@ import TimeLineChart from 'components/ui/TimeLineChart'
 const MAX_NINKI = 100
 
 const NINKI_BUILDERS = {
-	[ACTIONS.MUG.id]: 30,
-	[ACTIONS.ATTACK.id]: 6,
+	[ACTIONS.MUG.id]: 40,
+	[ACTIONS.MEISUI.id]: 40,
+	[ACTIONS.SPINNING_EDGE.id]: 8,
+	[ACTIONS.GUST_SLASH.id]: 8,
+	[ACTIONS.AEOLIAN_EDGE.id]: 8,
+	[ACTIONS.SHADOW_FANG.id]: 8,
+	[ACTIONS.ARMOR_CRUSH.id]: 8,
+	[ACTIONS.DEATH_BLOSSOM.id]: 8, // TODO - Confirm whether AoE weaponskills count
+	[ACTIONS.HAKKE_MUJINSATSU.id]: 8,
+	[ACTIONS.THROWING_DAGGER.id]: 8,
 }
 
 const NINKI_SPENDERS = {
 	[ACTIONS.HELLFROG_MEDIUM.id]: 80,
 	[ACTIONS.BHAVACAKRA.id]: 80,
-	[ACTIONS.TEN_CHI_JIN.id]: 80,
+	[ACTIONS.BUNSHIN.id]: 80,
 }
 
 export default class Ninki extends Module {
 	static handle = 'ninki'
 	static title = t('nin.ninki.title')`Ninki Timeline`
 	static displayMode = DISPLAY_MODE.FULL
 	static dependencies = [
-		'cooldowns',
 		'suggestions',
 	]
 
 	_ninki = 0
 	_ninkiHistory = []
 	_wasteBySource = {
 		[ACTIONS.MUG.id]: 0,
-		[ACTIONS.ATTACK.id]: 0,
+		[ACTIONS.MEISUI.id]: 0,
```

Comment:
out of scope for this pr; but would be cleaner to leave this blank and add as necessary (imo)